joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/ControlStructureTests.scala

```diff
@@ -59,7 +138,7 @@ class NewControlStructureTests extends JavaSrcCode2CpgFixture {
       assignment.order shouldBe 2
       assignment.typeFullName shouldBe "java.lang.String[]"
 
-      val (iterIdentifier, arrayAlloc) = assignment.argument.l match {
+      val (iterIdentifier, arrayInitializer) = assignment.argument.l match {
         case List(iterIdentifier: Identifier, arrayAlloc: Call) => (iterIdentifier, arrayAlloc)
```

Comment:
```suggestion
        case List(iterIdentifier: Identifier, arrayInitializer: Call) => (iterIdentifier, arrayInitializer)
```

nit: also rename the variable inside the match